### ISSUE TITLE
Proto removes comments when it modifies TOML files

### ISSUE THREAD

**Issue by Phault** [moonrepo/proto#424]
### What version?

0.31.1

### Which command?

proto pin

### What happened?

If you use comments in your .prototools, then after pinning a version using `proto pin` or adding a plugin with `proto plugin add` the comments are gone. This defeats one of the niceties of using TOML. :(

### Operating system?

MacOS

### Architecture?

arm64

**Comment by milesj** [comment]
Yeah it's on the roadmap. I need to switch over to `toml_edit` on the rust side.

**Comment by milesj** [comment]
This finally landed in v0.50
